components/ChatMessage.js

```diff
@@ -4,17 +4,18 @@ import { Content } from 'native-base';
 
 export class ChatMessage extends React.Component {
     render() {
-        const { message, user, isSelf } = this.props;
+        const { text, userId, createdTimestamp, currentUserId, currentUserName } = this.props;
+        // Need to fetch display name for users other than self, which will involve the restructuring of the chats/messages state, put the below just as a placeholder
```

Comment:
i think sonam updated the endpoint to return the display names and user ids of everyone in the chat

Reply:
Hmm, she had commented on another PR that I should get that stuff from the messages state--but yeah it does look like you can just get it from the chats state, I'm going to leave that as is for now in order to merge

Reply:
That's not what I said. See the PR in question: https://github.com/emote-chat/server/pull/21. I made that comment based on a conversation with Michele.